### PR TITLE
NWD_NWW homework

### PR DESCRIPTION
**Pull request by NikczemnyOkularnik**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @NikczemnyOkularnik

**Comment by ziobron** [comment]
⏱ 3 XP extra for @NikczemnyOkularnik for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
 
-int NWD(int lhs, int rhs) {
+int NWD(int lhs, int rhs) 
+{
     // TODO: Implement me :)
-    return -1;
+    //OK
+  
+    
+    int kwik;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:9]: Improper name.
Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
     // TODO: Implement me :)
-    return -1;
+    lhs = abs(lhs);
+    rhs = abs(rhs);
+    if(lhs == 0 || rhs == 0)
+    {
+        return 0;
+    }
+
+    return(lhs * rhs)/ NWD(lhs,rhs);
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:31]: Add space between operations like '+' '-' '!=' '==' '%' etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
+	
+    return lhs;
 }
 
 int NWW(int lhs, int rhs) {
     // TODO: Implement me :)
-    return -1;
+    lhs = abs(lhs);
+    rhs = abs(rhs);
+    if(lhs == 0 || rhs == 0)
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:26]: Add space after while, if, else etc...

File: homework/nwd-nnw/nwdNww.hpp (hunk 2 above)
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:31]: Add space after while, if, else, return etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**